views/ndt/referenced-by/referenced-by/unified_downloads_20201026.sql

```diff
@@ -0,0 +1,47 @@
+--
```

Comment:
I need change this and the legacy uploads file names.  The BQ UI treats objects ending in underscore-datecode as single partitions of a partitioned table.